graalvm/build.gradle

```diff
@@ -93,18 +93,18 @@ task buildGraal() {
  * rather than preserved. Graal fails to run with the resolved symlinks.
  */
 task tarGraal(type: Exec) {
-    dependsOn buildGraal
+    dependsOn(buildGraal)
 
     // Task details
     group = 'Other'
     description = 'Generate graalvm.tar file.'
 
     // Set the input to the folder in which the build is generated. We actually want to tar the only subdirectory from this.
-    inputs.dir(graalVMLatestDir)
+    inputs.file(graalBuildDummy)
```

Comment:
Can you leave a comment explaining why we cannot monitor the actual files that are used as input for this task?